lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -52,9 +52,9 @@ def handle_event_file(event_file)
         logger.debug("[foreman_ansible] - parsing event file #{event_file}")
         begin
           event = JSON.parse(File.read(event_file))
-          if (hostname = event.dig('event_data', 'host'))
+          if hostname_for_event(event)
             handle_host_event(hostname, event)
-          else
+          elsif event.key?('stdout')
```

Comment:
So if we have a play which runs on hosts A and B and which has a task which gets delegated to localhost for both, would this mean that outputs of both hosts will contain outputs for both delegated tasks?

Reply:
Yes, indeed, that's what would happen. I can try to investigate if there is some other indication as for what host this is running FOR, but unless I have any luck with that, I don't see much of better ways.

Reply:
From what I've seen, if a task is delegated, then `.event_data.res` has a `_ansible_delegated_vars` key, otherwise it is missing. Although I'm not sure how much we can rely on this.